src/adafruit_blinka/microcontroller/ft232h/url.py

```diff
@@ -0,0 +1,17 @@
+"""Support for getting the URL from the BLINKA_FT232H variable."""
+
+import os
+
+
+def get_ftdi_url():
+    """
+    Return the FTDI url to use. If BLINKA_FT232H starts with ftdi:, returns
+    that. Otherwise, returns a default value.
+    """
+
+    url = os.environ.get("BLINKA_FT232H", "1")
+
+    if url.startswith("ftdi:"):
+        return url
+
+    return "ftdi://ftdi:ft232h/1"
```

Comment:
Suggest moving this into `__init__.py` and then using a relative import in the other 3 .py files.